DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/mm-host/src/vector_add.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+// Buffer locations for MM Host interfaces
+static constexpr int BL0 = 0;
```

Comment:
do these need to be static?

Reply:
also naming convention in the code samples would probably require you to name these kBL0, kBL1, kBL2